pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -976,9 +978,15 @@ func (a *ApplicationBackupController) backupVolumes(backup *stork_api.Applicatio
 				return fmt.Errorf("error getting volume driver name: %v", err)
 			}
 			if driverName == volume.PortworxDriverName {
-				volumeInfos, err := driver.GetBackupStatus(backup)
+				batchCount := getBatchCountForGetBackupStatus()
+				volumeInfos, err := a.getAndUpdateBackupStatusInBatches(backup, driver, driverName, batchCount)
+				if err != nil {
+					return err
+				}
+				backup.Status.Volumes = volumeInfos
```

Comment:
Do we need this update here as in the caller of `getAndUpdateBackupStatusInBatches()` update is done for each vol status fetched. Looking if this can be optimized here.